<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPSinkGUI</class>
 <widget class="RollupWidget" name="UDPSinkGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>342</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>UDP Sample Sink</string>
  </property>
  <property name="statusTip">
   <string>UDP Sample Sink</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>380</width>
     <height>142</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="ChannelPowerLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="channelPower">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Channel output power (dB)</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>-100.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="channelPowerUnits">
        <property name="text">
         <string> dB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="DeltaFrequencyLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="deltaFrequencyLabel">
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Df</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ValueDialZ" name="deltaFrequency" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Demod shift frequency from center in Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deltaUnits">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>26</red>
              <green>26</green>
              <blue>26</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>26</red>
              <green>26</green>
              <blue>26</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>118</red>
              <green>118</green>
              <blue>117</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Hz</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="inputPower">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Input power (dB) to which squelch applies</string>
        </property>
        <property name="text">
         <string>-100.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="SampleRateLayout">
      <item>
       <widget class="QLabel" name="sampleRateLabel">
        <property name="text">
         <string>SRout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sampleRate">
        <property name="toolTip">
         <string>Output sample rate (S/s)</string>
        </property>
        <property name="text">
         <string>48000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="RFBandwidthLayout">
      <item>
       <widget class="QLabel" name="rfBandwidthLabel">
        <property name="text">
         <string>RFBW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rfBandwidth">
        <property name="toolTip">
         <string>Signal bandwidth (Hz)</string>
        </property>
        <property name="text">
         <string>32000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="applyBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Apply text input and/or samples format</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="AddressLayout">
      <item>
       <widget class="QLabel" name="outputUDPAddressLabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputUDPAddress">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Destination UDP address</string>
        </property>
        <property name="inputMask">
         <string>000.000.000.000</string>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="outputUDPPortLabel">
        <property name="text">
         <string>P</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputUDPPort">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Destination UDP port</string>
        </property>
        <property name="inputMask">
         <string>00000</string>
        </property>
        <property name="text">
         <string>9998</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="FormatLayout">
      <item>
       <widget class="QLabel" name="formatLabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Fmt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sampleFormat">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Samples format</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>I/Q 16bits</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>I/Q 24bits</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NFM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NFM Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LSB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LSB Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USB Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AM Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AM !DC Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AM BPF Mono</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="inputUDPAudioPortLabel">
        <property name="text">
         <string>Au</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputUDPAudioPort">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Audio input UDP port</string>
        </property>
        <property name="inputMask">
         <string>00000</string>
        </property>
        <property name="text">
         <string>9997</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="AudioPortLayout">
      <item>
       <widget class="QLabel" name="fmDevLabel">
        <property name="text">
         <string>FMd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fmDeviation">
        <property name="toolTip">
         <string>FM deviation in Hz (for S16LE NFM format)</string>
        </property>
        <property name="text">
         <string>2500</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="agc">
        <property name="toolTip">
         <string>Toggle AGC (only for AM and SSB)</string>
        </property>
        <property name="text">
         <string>AGC</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="audioActive">
        <property name="toolTip">
         <string>Toggle audio input</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/sound_off.png</normaloff>
          <normalon>:/sound_on.png</normalon>:/sound_off.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="audioStereo">
        <property name="toolTip">
         <string>Toggle mono/stereo audio input</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/mono.png</normaloff>
          <normalon>:/stereo.png</normalon>:/mono.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="VolumeLayout">
      <item>
       <widget class="QLabel" name="volumeLabel">
        <property name="text">
         <string>Vol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volume">
        <property name="toolTip">
         <string>Audio volume</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="volumeText">
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>20</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gainLabel">
        <property name="text">
         <string> Gain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="gain">
        <property name="toolTip">
         <string>Output linear gain</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gainText">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Output linear gain</string>
        </property>
        <property name="text">
         <string>00.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="SquelchLayout">
      <item>
       <widget class="QLabel" name="squelchLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Sq</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="squelch">
        <property name="toolTip">
         <string>Power squelch threshold (dB)</string>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>-60</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="squelchText">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Power squelch threshold (dB)</string>
        </property>
        <property name="text">
         <string>-00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="squelchGate">
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Squelch gate (ms)</string>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="squelchGateText">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Squelch gate (ms)</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="spectrumBox" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>231</width>
     <height>156</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Channel Spectrum</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="GLSpectrum" name="glSpectrum" native="true">
      <property name="font">
       <font>
        <family>Liberation Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GLSpectrumGUI" name="spectrumGUI" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrum</class>
   <extends>QWidget</extends>
   <header>gui/glspectrum.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLSpectrumGUI</class>
   <extends>QWidget</extends>
   <header>gui/glspectrumgui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
